### PR TITLE
Add simple bash script to upload a msadoc.json file

### LINKED ISSUE #3: Simple CLI
## User Story
* As a msadoc user,
* I want to use a CLI program or script to upload the latest `msadoc.json` of my service
* in order to use it from my PC or in CI/CD pipelines.

## Acceptance criteria
* A shell script to upload the `msadoc.json` file
* Parameters: `IP_SERVER`, `MSADOC_FILE_PATH`, `API_KEY` 
* Manual tests should suffice
* Updated documentation

### PR DESCRIPTION
**Pull request by georg-schwarz**
Closes #3

### CODE REVIEW

File: cli/msadoc-upload.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+if [ -z "$apikey" ]
+then
+   echo "ApiKey parameter is missing";
+   printHelp
+fi
+
+if [ -z "$filepath" ]
+then
+   echo "[INFO] ApiKey parameter defaults to './msadoc.json'";
```
Comment (MBuchalik) [inline cli/msadoc-upload.sh:50]:
```suggestion
   echo "[INFO] Filepath parameter defaults to './msadoc.json'";
```

File: cli/msadoc-upload.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+response_code="$(
+    curl -X POST "${server}/service-docs" \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer ${apikey}" \
+    -d @"$filepath" \
+    --write-out '%{http_code}' \
+    --silent \
+    --output /dev/null
+)"
+echo "[INFO] Finished with status code ${response_code}!"
```
Comment (MBuchalik) [inline cli/msadoc-upload.sh:66]: Should we maybe exit with `1` if uploading failed? (I am not a bash script expert - maybe, this is already how this script behaves?) My idea is that exiting with `1` will, on many setups, make CI fail. Which will draw attention to the failing upload.

Also, an empty newline is missing at the end of this file.

**Review by MBuchalik — COMMENTED** [review]
I am really no bash script expert, so my review is rather limited. But in general, it looks good to me 👍

File: cli/msadoc-upload.sh (hunk 2 above)
Comment (georg-schwarz) [reply cli/msadoc-upload.sh:66]: Agreed, moved into separate issue.

**Pull request merged**